src/apiCalls.js

```diff
@@ -0,0 +1,20 @@
+export const fetchRooms = () => {
+  return fetch("http://localhost:3001/api/v1/rooms")
+    .then(response => response.json())
+    .then(data => data.rooms)
+    .catch(err => console.log('Oops! Something went wrong!', err))
+}
+
+export const fetchBookings = () => {
+  return fetch("http://localhost:3001/api/v1/bookings")
+    .then(response => response.json())
+    .then(data => data.bookings)
+    .catch(err => console.log('Oops! Something went wrong!', err))
+}
+
+export const fetchCustomers = () => {
+  return fetch("http://localhost:3001/api/v1/customers")
+    .then(response => response.json())
+    .then(data => data.customers)
+    .catch(err => console.log('Oops! Something went wrong!', err))
+}
```

Comment:
These look good to me! You might consider adding the one for a single user with the id interpolated but just a suggestion (not sure it has to be there).